Extending Docker image to have Bex screenshot extension for Behat

toni-kolev opened the pull request:
Adding `bex/behat-screenshot` to composer.

toni-kolev committed:
Extending Docker image to have Bex screenshot extension for Behat

lmakarov commented:
@toni-kolev can you please also provide an example of how you are using `bex/behat-screenshot`?

toni-kolev committed:
Adding Example configurations for Bex Screenshot

toni-kolev committed:
Updating README to contain information about Bex extension

toni-kolev commented:
@lmakarov I have added example configuration in the `behat.yml` and updated the `README` with some information about the extension. Let me know if there's anything else that is needed.

lmakarov commented on example/behat.yml:
```diff
@@ -48,6 +48,12 @@ default:
+    Bex\Behat\ScreenshotExtension:
+      image_drivers:
+        local:
+          screenshot_directory: "%paths.base%/<change_screenshots_folder_name>/"
```
Can we assume a default folder name, which works out of the box (is created automatically)?

toni-kolev replied:
If you don't provide a folder screenshots are stored in `/tmp/behat-screenshot/`

![image](https://user-images.githubusercontent.com/5762835/58196386-db8df200-7cd2-11e9-9ee3-2faa27365ed1.png)

If you provide a folder path and the folder does not exist, it is created.

lmakarov replied:
If it's just set to `%paths.base%/screenshots/` by default, where will the screenshots go?

toni-kolev replied:
`%paths.base%` refers to the base of Behat. This is the folder which contains the `behat.yml` and the `features` folder. In other words the `screenshots` folder will be sibling of the `features` folder.

![image](https://user-images.githubusercontent.com/5762835/58200923-9373cd00-7cdc-11e9-9fdc-0300f19a64a4.png)

lmakarov replied:
> In other words the screenshots folder will be sibling of the `features` folder.

That sounds like a fine place to place the `screenshots` folder, so let's just go with that. This way users won't have to edit anything out of the box.

toni-kolev replied:
I agree. Changes to `behat.yml` are commited.

toni-kolev committed:
Setting screenshots as default folder

lmakarov commented:
Thanks!
